quarkus-integration/quarkus-benchmark/integration-test/src/main/java/ai/timefold/solver/quarkus/benchmark/it/TimefoldBenchmarkTestResource.java

```diff
@@ -38,10 +41,25 @@ public String benchmark() {
                 new TestdataListValueShadowEntity("ccc")));
         PlannerBenchmark benchmark = benchmarkFactory.buildPlannerBenchmark(planningProblem);
         try {
+            // We have tests that share the benchmark folder,
+            // and we need to ensure the benchmark is executed in a clean folder.
+            deleteAllFolders(((DefaultPlannerBenchmark) benchmark).getBenchmarkDirectory());
```

Comment:
Now that I think of it... wouldn't this better work in `@BeforeEach` in a test?

We can even make JUnit use resource locking to avoid multiple tests over the same directory. (Although that is brittle, because that would require all such tests to be annotated, and therefore one non-annotated test breaks all of this. So just an idea.)